pxr/imaging/plugin/hdRpr/rprApiAov.h

```diff
@@ -113,100 +212,123 @@ class HdRprApiColorAov : public HdRprApiAov {
             return !(*this == lhs);
         }
     };
+
+    struct UpscaleParams {
+        bool enable;
+
+        enum class Mode {
+            Good,
+            Best,
+            Fast
+        };
+
+        Mode mode;
```

Comment:
Why don't we just store RIF_AI_UPSCALE_MODE directly instead of introducing one more enum?

Reply:
I'd prefer strongly typed enums instead of defines because of their safety.
You think its better to use uint to store this?

Reply:
There is no other use of this strongly typed enum than converting from TfToken to RIF define. So what's the point, what do we get from this intermediary?

Reply:
I can see a few pros of using this enum:
- Type safety, it's harder to pass invalid value of enum compared to using int
- Easier for new people to understand what this field mean, no need to search for usage what exact RIF values are assigned to the int